module-apps/apps-common/widgets/TimeSetFmtSpinner.cpp

```diff
@@ -296,14 +296,14 @@ namespace gui
         period == Period::AM ? fmt->set_value(utils::time::Locale::getAM())
                              : fmt->set_value(utils::time::Locale::getPM());
     }
-    auto TimeSetFmtSpinner::getHour24Format() const noexcept -> std::chrono::hours
+    auto TimeSetFmtSpinner::getHour24Format() const noexcept -> int
     {
-        using namespace utils::time;
-        if (timeFormat == Locale::TimeFormat::FormatTime24H) {
-            return std::chrono::hours{timeSetSpinner->getHour()};
+        if (timeFormat == utils::time::Locale::TimeFormat::FormatTime24H) {
+            return timeSetSpinner->getHour();
         }
-
-        return date::make24(std::chrono::hours{timeSetSpinner->getHour()}, isPM());
+        using namespace utils;
```

Comment:
This `using namespace utils;` seems to be not needed, you can remove it or change it to `using namespace utils::time;`, place at the beginning of the method and compare `timeFormat == Locale::TimeFormat::FormatTime24H`